tests/testthat/testSel3D.R

```diff
@@ -0,0 +1,55 @@
+# testSel3D.R
+#
+#
+# Structure your tests according to the three principles expressed below:
+#    test_that("parameter errors are correctly handled", { ... })
+#    test_that("a sane input gives an expected output", { ... })
+#    test_that("a corrupt input does not lead to corrupted output", { ...})
+
+
+# ==== BEGIN SETUP AND PREPARE =================================================
+#OLOG <- as.character(getOption("rete.logfile"))   # save original logfile name
+#logFileName(fPath = tempdir(), setOption = TRUE)  # make tempdir() the log dir
+#NL <- .PlatformLineBreak()
+# ==== END SETUP AND PREPARE ===================================================
+
+
+
+
+#test_that("parameter errors are correctly handled", {
+  # Try missing input sequence
+  # Try extraneous parameters
+  # Note: we're not to attempt to avoid all kinds of abuse from users
```

Comment:
It might be easier to refer to the R code that has been merged into the rete package when filling out your function documentation. Don't "reinvent the wheel".